src/content_sao.h

```diff
@@ -121,6 +121,7 @@ class LuaEntitySAO : public UnitSAO
 	v3f getAcceleration();
 
 	void setTextureMod(const std::string &mod);
+	std::string getTextureMod();
```

Comment:
This function should be const, although I disagree with a pointer parameter - we should avoid returning things through parameters unless absolutely performance critical